### ISSUE TITLE
Apk pure updates not working 

### ISSUE THREAD

**Issue by mubaidr** [rumboalla/apkupdater#577]
Apk pure updates does not install

**Comment by je-vv** [comment]
Not just updates, installing.  It seems it can't even download the APKs.

BTW, it does show pending updates and notifies them.  I initially thought it was about wrong APKs, but this has been consistent for the past week, and maybe more.

In my case, using the IzzyOnDroid apkupdate which is up to date on 3.0.3 version.

Maybe it's time for a new release, last one dates to January...  There are several commits from January till Today and maybe head of main/master is working just fine, which would indicate the 3.0.3 release got somehow broken.

**Comment by je-vv** [comment]
BTW @rumboalla to me it looks like apkupdater is no longer connecting correctly to apkpure neither apkmirror (I stopped using apkmirror since their multi-apks no longer install on my LOS), it can't find several apps on them, and on apkpure there's the ones that show up don't show icons...  I'm under the impression that apkupdater's functionality is gradually degrading for its 3.0.3 release.

Thanks a lot !

**Comment by archon810** [comment]
> I stopped using apkmirror since their multi-apks no longer install on my LOS)

Can you please expand on this? What doesn't install, what reason is being shown, etc? This should not happen.

**Comment by je-vv** [comment]
> > I stopped using apkmirror since their multi-apks no longer install on my LOS)
> 
> Can you please expand on this? What doesn't install, what reason is being shown, etc? This should not happen.

Apkmirror doesn't allow installing other than with its own app or through its web page, which is what apk upupdater and its now defunct clone apkgrabber do, just take the user to the web page of the package for the particular version to download the APK.  But on my last trials, when downloading their APKs and trying to install them, with little to no exceptions, the APKs refused to install unless using Apkmirror's own package manager APP, given their own multi APKs  implementation.  After several APKs failing like that I stopped using Apkmirror.

Apkpure was working flawlessly instead, and I was able to use apkupdater to download and install directly from Apkpure, but now when I tap on "install" the APK never gets downloaded.  Not sure if this is because of some wrong doing by Apkpure, or what else.  This actually makes it really hard for me to install/upgrade anything from Apkmirror (which I discarded some time back) and now from Apkpure, leaving apkupdater of no use for me (Aptoid barely offers me any upgrade).

Not sure if I answered your question well enough, and if I didn't, please let me know.

**Comment by iamLazyCode** [comment]
does apps from github install ? i have a beta version of an app installed and now there's a stable release from github but that doesnt install

any help ?

**Comment by je-vv** [comment]
@iamLazyCode I don't know, but if it doesn't, there's no issue, since you can install the app with your browser, or you could just download the apk with your browser and install it through `Files` or any other file manager, as long as you grant the browser, the file manager or whatever other app the permission as 3rd party installers (that'll be asked to you any ways when trying to open the apk).  So I'd say that shouldn't be a big concern, but perhaps what you're actually indicating is that installing from github is not working eiehtr for you?

**Comment by iamLazyCode** [comment]
> @iamLazyCode I don't know, but if it doesn't, there's no issue, since you can install the app with your browser, or you could just download the apk with your browser and install it through `Files` or any other file manager, as long as you grant the browser, the file manager or whatever other app the permission as 3rd party installers (that'll be asked to you any ways when trying to open the apk).  So I'd say that shouldn't be a big concern, but perhaps what you're actually indicating is that installing from github is not working eiehtr for you?

Yeah i manually updated via file manager but just to report i commented...the soul purpose of this tool is to install updates at once place...and via github i jist tried it just gave me app failed to install error

**Comment by je-vv** [comment]
Yeah, I'm sad apkupdater, and its fork apkgrabber (actually archived already) can not be used anymore it seems then

**Comment by iamLazyCode** [comment]
> Yeah, I'm sad apkupdater, and its fork apkgrabber (actually archived already) can not be used anymore it seems then

Hmmm thanks for the info and help bro i hope there may be any app that may support app updates like this in future

**Comment by je-vv** [comment]
Well, I only use apkupdater for proprietary stuff not found on f-droid (official + non official), since I don't like aurora, and it's slow to grab upgrades, and depends on Google Play, which I really dislike.  If you want open source apks released on github for example, you can try obtainium, which I really don't like much, but many people endorse it.  But it's not a replacement for proprietary apks available on apkpure/apkmirror for example (I haven't seen anything close to apkupdater/apkgrabber for these) which one might still need to install, for those one still need aurora store, or apkupdater/apkgrabber getting the apks from sources different than Google, but what apkupdater offers for apkpure for example has no replacement AFAIK

**Comment by iamLazyCode** [comment]
> Well, I only use apkupdater for proprietary stuff not found on f-droid (official + non official), since I don't like aurora, and it's slow to grab upgrades, and depends on Google Play, which I really dislike. If you want open source apks released on github for example, you can try obtainium, which I really don't like much, but many people endorse it. But it's not a replacement for proprietary apks available on apkpure/apkmirror for example (I haven't seen anything close to apkupdater/apkgrabber for these) which one might still need to install, for those one still need aurora store, or apkupdater/apkgrabber getting the apks from sources different than Google, but what apkupdater offers for apkpure for example has no replacement AFAIK

yes i used obtanium on phones and aurora is pain on latest updates when it disabled to set location for downloaded apps...can apkupdater set custom path for downloaded apps ? i was just looking for a solid app for my tv

**Comment by TPS** [comment]
Of course, this'd work if my system adblock/firewall on the 12 device wasn't _also_ blocking `winudf.com`, since that's where APKU keeps its files.
